sections/transactions.js

```diff
@@ -73,12 +65,8 @@ export default [
           .into('catalogues')
           .transacting(trx)
           .then(function(ids) {
```

Comment:
In this query spamming queries to driver buffer is fixed, but .transacting(trx) was not changed.

Reply:
Isn't whole idea of second example to showcase supported .transacting() syntax? First example shows the trx() one.

Reply:
dunno, maybe :) I didn't look the whole picture

Reply:
to me its fine like this if you think its good

Reply:
Now I only need to complete the actual PR with feature 😂